mach

```diff
@@ -160,6 +160,11 @@ def build(args):
         - disable
         - coverage
     """
+
+    # Create the build folder.
+    if not os.path.exists("build"):
```

Comment:
Do we need to do this now before CMake does it?

Reply:
Yes. `mach` does a couple of checks before starting `cmake` such as the dependency check. The check creates a lock-file and this file has to go into the build folder (which must exist).